lua/nvim-treesitter/install.lua

```diff
@@ -555,6 +555,7 @@ end
 function M.setup_auto_install()
   vim.api.nvim_create_autocmd("FileType", {
     pattern = { "*" },
+    group = vim.api.nvim_create_augroup('NvimTreesitter-auto_install', { clear = true }),
```

Comment:
stylua doesn't like the quotes and I don't like the name.

Reply:
I've fixed the stylua issue. The augroup name is chosen after `NvimTreesitter-<module_name>` structure. Do you have any other suggestion? I can't use `NvimTreesitter` because it's defined in `plugin/nvim-treesitter.lua`.

Reply:
The modules will be removed in 1.0.
You can use `group = "NvimTreesitter"`.

Reply:
If we do not clear an own group upon `M.setup_auto_install()`, we still have the same problem: Call `setup_auto_install()` multiple times (from user config) and then multiple FileType events.

```diff
 function M.setup_auto_install()
   vim.api.nvim_create_autocmd("FileType", {
     pattern = { "*" },
-    group = vim.api.nvim_create_augroup('NvimTreesitter-auto_install', { clear = true }),
+    group = "NvimTreesitter",
     callback = function()
       ...
     end,
  })
 end
```

If we use `group = vim.api.nvim_create_augroup('NvimTreesitter', { clear = true })`, it will clear other autocmds under the group defined in `plugin/nvim-treesitter.lua`. So I think we should use a different augroup; what do you think?